### PR TITLE
Add new slap messages

### PR DESCRIPTION
**Pull request by YakoYakoYokuYoku**
Adds new .slap messages just to commemorate April's Fools. :clown_face: :clown_face: :clown_face:

### CODE REVIEW

File: slap_plugin.go [hunk 1]
```diff
@@ -30,13 +30,18 @@ func init() {
 		"destroys $USER",
 		"discombobulates $USER",
 		"does far worse, taking $USER's system and installing Windows",
+		"drinks $USER coffee",
+		"eats $USER pizza",
```
Comment (EbonJaeger) [inline slap_plugin.go]: These should be `$USER's` in order to be valid English. :P

Comment (YakoYakoYokuYoku) [reply slap_plugin.go]: Now It's solved c:

**Review by EbonJaeger — APPROVED** [review]
This doesn't mean anything, I just like hitting the buttons. :P

File: slap_plugin.go [hunk 2]
```diff
@@ -30,13 +30,18 @@ func init() {
...
 		"does far worse, taking $USER's system and installing Windows",
+		"drinks $USER's coffee",
+		"eats $USER's pizza",
 		"gives $USER a splinter",
 		"just looks at $USER with disappointment",
+		"launches $USER to space",
 		"opts to not slap $USER today, but rather gives them a cookie",
 		"punches $USER",
 		"rejects $USER's patches",
+		"sentences $USER to commit seppuku",
```
Comment (JoshStrobl) [inline slap_plugin.go]: nonono

**Review by JoshStrobl — CHANGES_REQUESTED** [review]

File: slap_plugin.go (hunk 2 above)
Comment (YakoYakoYokuYoku) [reply slap_plugin.go]: Now the user has a less luckier fate.

**Review by JoshStrobl — APPROVED** [review]

**Comment by JoshStrobl** [comment]
Cheers, will need to cook up a new Narwhal for this package update :)

**Pull request merged**